WalletWasabi.Fluent/Views/Wallets/Send/TransactionSummary.axaml

```diff
@@ -35,7 +35,7 @@
 
     <!-- Recipient -->
     <c:PreviewItem Icon="{StaticResource person_regular}"
-                   Label="will be sent to"
+                   Label="receiver"
```

Comment:
It should be the same as the title of the dialog where they enter the label.

Reply:
Done.